packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeFeeRegistry.sol

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "./AirnodeFeeRegistryRolesWithManager.sol";
+import "./interfaces/IAirnodeFeeRegistry.sol";
+
+/// @title Contract for storing the price of a airnode endpoints
```

Comment:
```suggestion
/// @title Contract for storing the price of an airnode endpoints
```

Reply:
changed a bunch of comments can have a look again